### PR TITLE
Add phase engine and tone mapping

### PR DESCRIPTION
**Pull request by SatjaSivcev**

### CODE REVIEW

File: source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cs [hunk 1]
```diff
@@ -32,7 +33,9 @@ static void Main()
                                                                                                   Strength = 0.4 },
                                                                                    Removal = { Enabled = true,
                                                                                                Threshold = 0.5 } } } },
-                               Color = { Balance = { Red = 1.0, Green = 1.0, Blue = 1.0 }, Gamma = 1.0 } }
```
Comment (chrisasc) [inline source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cs:35]: Just curious, is this the preferred structure of the code from CI? 

Personally I think something like this is more readable:
                               Color = { 
                               Balance = { Red = 1.0, Green = 1.0, Blue = 1.0 },
                               Gamma = 1.0,
                               Experimental = { ToneMapping = { Enabled = ToneMappingEnabledOption.HdrOnly } } } 
                               }

**Review by chrisasc — APPROVED** [review]
lgtm, i have a single questions that I no show-blocker to me

File: source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cs (hunk 1 above)
Comment (SatjaSivcev) [reply source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cs:35]: We don't check for the code structure in the C# CI.

Comment (SatjaSivcev) [reply source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cs:35]: I changed it to this now:

`            var settings = new Zivid.NET.Settings()
            {
                Experimental = { Engine = Zivid.NET.Settings.ExperimentalGroup.EngineOption.Phase },
                Processing = { Filters = { Smoothing = { Gaussian = { Enabled = true, Sigma = 1.5 } },
                                           Noise = { Removal = { Enabled = true, Threshold = 7.0 } },
                                           Outlier = { Removal = { Enabled = true, Threshold = 5.0 } },
                                           Reflection = { Removal = { Enabled = true } },
                                           Experimental = { ContrastDistortion = { Correction = { Enabled = true,
                                                                                                  Strength = 0.4 },
                                                                                   Removal = { Enabled = true,
                                                                                               Threshold = 0.5 } } } },
                               Color = { Balance = { Red = 1.0, Green = 1.0, Blue = 1.0 },
                                         Gamma = 1.0,
                                         Experimental = { ToneMapping = { Enabled = ToneMappingEnabledOption.HdrOnly } } } }
            };`

Comment (SatjaSivcev) [reply source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cs:35]: I changed the formating to something I think is better now.

**Pull request merged**